content/applications/finance/accounting/taxes.rst

```diff
@@ -72,23 +72,40 @@ The **tax name** is displayed for backend users in the :guilabel:`Taxes` field i
 Tax computation
 ~~~~~~~~~~~~~~~
 
-- **Group of Taxes**
+The :guilabel:`Tax Computation` field determines how the tax amount is computed from the sales
+price. The following options are available:
+
+- :ref:`taxes/computation/group-of-taxes`: a combination of several other taxes
+- :ref:`taxes/computation/fixed`: a fixed amount
+- :ref:`taxes/computation/percentage-of-price`: a percentage of the tax-excluded sales price
+- :ref:`taxes/computation/percentage-of-price-tax-included`: a percentage of the tax-included total
+- :ref:`taxes/computation/python-code`: a custom user-defined formula.
+
+.. _taxes/computation/group-of-taxes:
+
+Group of Taxes
```

Comment:
```suggestion
Group of taxes
```
We use [sentence case](https://www.odoo.com/documentation/18.0/contributing/documentation/content_guidelines.html#document-structure) for headings, even though this contradicts another [guideline](https://www.odoo.com/documentation/18.0/contributing/documentation/content_guidelines.html#capitalization), which is to refer to Odoo's UI elements as they appear. In any case, sentence case takes precedence :)
You will need to adapt the references above (e.g., ```:ref:`Group of Taxes <taxes/computation/group-of-taxes>` ``` to follow the second guidelines :)

Reply:
This makes sense, thank you very much!